apps/passport-client/public/index.html

```diff
@@ -4,10 +4,23 @@
     <title>Zuzalu Passport</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, user-scalable=no" />
-    <link rel="stylesheet" href="/global.css" />
-    <script defer src="/js/index.js"></script>
+    <link rel="stylesheet" href="global.css" />
+    <script defer src="js/index.js"></script>
+    <link rel="manifest" href="manifest.json" />
+    <script>
+      if ('serviceWorker' in navigator) {
+        window.addEventListener('load', () => {
+          navigator.serviceWorker.register('service-worker.js')
+            .then((registration) => {
+              console.log('Service Worker registered with scope:', registration.scope);
+            }, (err) => {
+              console.error('Service Worker registration failed:', err);
+            });
+        });
+      }
```

Comment:
code looks reasonable to me!

rather than putting it directly into `index.html` though, put it into its own file (eg `init-service-worker.ts`) and call it from the main typescript entry point